sympy/physics/quantum/cg.py

```diff
@@ -226,16 +226,16 @@ def _pretty(self, printer, *args):
         top = printer._print_seq((self.j3, self.m3), delimiter=',')
 
         pad = max(top.width(), bot.width())
-        bot = prettyForm(*bot.left(' '))
-        top = prettyForm(*top.left(' '))
+        bot = bot.left(' ')
+        top = top.left(' ')
 
         if not pad == bot.width():
-            bot = prettyForm(*bot.right(' '*(pad - bot.width())))
+            bot = bot.right(' '*(pad - bot.width()))
         if not pad == top.width():
-            top = prettyForm(*top.right(' '*(pad - top.width())))
-        s = stringPict('C' + ' '*pad)
-        s = prettyForm(*s.below(bot))
-        s = prettyForm(*s.above(top))
+            top = top.right(' '*(pad - top.width()))
```

Comment:
@oscarbenjamin the most disruptive changes I need to pass all the tests are like this one: instead of calling the constructor of `prettyForm` over the result of `right` (and the other methods) I just pick the output of the method.